st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -43,6 +43,9 @@
 ]
 
 LOG = logging.getLogger(__name__)
+LOG_PATH = '/var/log/st2/'
```

Comment:
Seems odd to have to set LOG_PATH here, is there a centrally defined path you can reference here?